src/App.tsx

```diff
@@ -15,36 +18,109 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+const sortByAlphabet = (data: Goods) => {
+  return data.sort((a, b) => a.localeCompare(b));
+};
+
+const sortByLength = (data: Goods) => {
+  return data.sort((a, b) => a.length - b.length || a.localeCompare(b));
+};
+
+enum Sorting {
+  Initial = '',
+  Alphabet = 'ALPHABET',
+  Length = 'LENGTH',
+}
+
+type Props = {
+  reversed?: boolean;
+  sorted?: Sorting;
+};
+
+export const App: React.FC<Props> = ({ reversed = false, sorted = '' }) => {
```

Comment:
```suggestion
export const App = () => {
```
You don't need to accept props in this component